### PR TITLE
Migrate WebSub module to new runtime APIs

### PR DESCRIPTION
**Pull request by Bhashinee**
## Purpose
$subject

### CODE REVIEW

File: websub-native/src/main/java/org/ballerinalang/net/websub/nativeimpl/HubNativeOperationHandler.java [hunk 1]
```diff
@@ -60,9 +58,13 @@
      * @param webSubHub respective hub instance
      * @return an array of available topics
      */
-    public static ArrayValue getAvailableTopics(BObject webSubHub) {
+    public static BArray getAvailableTopics(BObject webSubHub) {
         String[] topics = Hub.getInstance().getTopics();
-        return new ArrayValueImpl(StringUtils.fromStringArray(topics));
```
Comment (warunalakshitha) [inline websub-native/src/main/java/org/ballerinalang/net/websub/nativeimpl/HubNativeOperationHandler.java:65]: we can directly return StringUtils.fromStringArray(topics) which provides BArray

File: websub-native/src/main/java/org/ballerinalang/net/websub/WebSubEmptyCallableUnitCallback.java [hunk 2]
```diff
@@ -33,6 +34,8 @@ public void notifySuccess() {
 
     @Override
     public void notifyFailure(io.ballerina.runtime.api.values.BError error) {
-        ErrorHandlerUtils.printError("error: " + error.getPrintableStackTrace());
```
Comment (warunalakshitha) [inline websub-native/src/main/java/org/ballerinalang/net/websub/WebSubEmptyCallableUnitCallback.java:36]: why we can't use error.printStackTrace() ?

Comment (Bhashinee) [reply websub-native/src/main/java/org/ballerinalang/net/websub/WebSubEmptyCallableUnitCallback.java:36]: I have used the `error.printStackTrace() `. But for that, we have to pass a `PrintWriter` right?

Comment (warunalakshitha) [reply websub-native/src/main/java/org/ballerinalang/net/websub/WebSubEmptyCallableUnitCallback.java:36]: error.printStackTrace() should also work without printwriter

File: websub-native/src/main/java/org/ballerinalang/net/websub/BallerinaWebSubConnectorListener.java [hunk 3]
```diff
@@ -375,9 +372,9 @@ private BString getParamStringValue(BMap<BString, Object> params, BString key) {
             return StringUtils.fromString("");
```
Comment (warunalakshitha) [inline websub-native/src/main/java/org/ballerinalang/net/websub/BallerinaWebSubConnectorListener.java:372]: If it is constant value we are passing to fromString method, we can replace them with BString constant so we do not need to create Bstring everytime.

**Review by chamil321 — APPROVED** [review]

**Pull request merged**